libsignal-service/src/account_manager.rs

```diff
@@ -109,6 +109,10 @@ impl<Service: PushService> AccountManager<Service> {
             pre_keys: pre_key_entities,
             signed_pre_key: signed_pre_key.into(),
             identity_key: identity_key_pair.public(),
+            last_resort_key: Some(PreKeyEntity {
+                key_id: 0x7fffffff,
+                public_key: "NDI=".into(),
+            }),
```

Comment:
This method still needs a way to signify that we want this `Some` or `None`; primary registrations needs to be able to put this at `None` (I'm using this method in WF).